src/components/Router.tsx

```diff
@@ -19,7 +19,7 @@ import { Routes, Route } from 'react-router-dom';
 import { rootRouteRef } from '../plugin';
 import FirebaseFunctionsPage from './FirebaseFunctionsPage';
 import { FIREBASE_FUNCTION_IDS } from '../hooks/useFunctionIds';
-import { WarningPanel } from '@backstage/core';
+import { MissingAnnotationEmptyState, WarningPanel } from '@backstage/core';
```

Comment:
I would suggest to delete WarningPanel from import as it is not used anymore and might be deleted in the future releases of backstage.

Reply:
Done